src/management/projects.ts

```diff
@@ -145,7 +222,8 @@ export async function getFirebaseProject(projectId: string): Promise<FirebasePro
   } catch (err) {
     logger.debug(err.message);
     throw new FirebaseError(
-      `Failed to get Firebase project ${projectId}. See firebase-debug.log for more info.`,
+      `Failed to get Firebase project ${projectId}` +
+        ". Please make sure the project exists and your account has permission to access it.",
```

Comment:
super-nit: Please put the leading `. ` (including the space) on the previous line